docs/DeveloperGuide.md

```diff
@@ -641,8 +642,87 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     * 2a1. iVolunteer requests for the correct command with valid event id.
     * 2a2. User enters correct command.
     Steps 2a1-2a2 are repeated until the data entered is correct.
+  
     Use case resumes at step 3.
 
+**Use case UCE05: Add a volunteer into an event**
+
+**MSS**
+
+1.  User <u>lists all volunteering events (UCE02)</u>.
+2.  User <u>lists all volunteers (UCV02)</u>.
+3.  User requests to add a specific volunteer in the volunteer list into a specific event in the event list.
+4.  iVolunteer adds the volunteer into the event.
+
+    Use case ends.
+
+**Extensions**
+
+* 3a. Invalid Command Word.
+    * 3a1. iVolunteer prompts Volunteer Coordinator to provide a valid command.
+
+      Use case resumes from step 3.
+
+* 3b. Invalid event id or volunteer id or both.
+    * 3b1. iVolunteer requests for the correct command with valid event id and valid volunteer id.
+    * 3b2. User enters correct command.
+      Steps 3b1-3b2 are repeated until the data entered is correct.
+
+      Use case resumes at step 4.
+
+**Use case UCE06: List out all volunteers in an event**
+
+**MSS**
+
+1.  User <u>lists all volunteering events (UCE02)</u>.
+2.  User requests to list out all volunteers in the specific event.
+3.  iVolunteer list out all volunteers in the events.
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. Invalid Command Word.
+    * 2a1. iVolunteer prompts Volunteer Coordinator to provide a valid command.
+
+      Use case resumes from step 2.
+
+* 2b. Invalid event id.
+    * 2b1. iVolunteer requests for the correct command with valid event id.
+    * 2b2. User enters correct command.
+      Steps 2b1-2b2 are repeated until the data entered is correct.
+  
+      Use case resumes at step 3.
+
+* 2c. There are no events to list.
+
+  Use case ends.
+
+**Use case UCE07: Remove a volunteer from an event**
+
+**MSS**
+
+1.  User <u>lists all volunteering events (UCE02)</u>.
+2.  User <u>lists all volunteers (UCV02)</u>.
+3.  User requests to remove a specific volunteer in the volunteer list into a specific event in the event list
+4.  iVolunteer removes the volunteer in the event.
```

Comment:
"in" can be changed to "from" for better grammar!

Reply:
ok boss!